---
title: Curriculum Vitae
layout: default
---
<!-- For altmetric badges -->
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

<h3>Education</h3>
<dl class="cv">
<dt>2019</dt>
<dd><strong>PhD Cell and Molecular Biology</strong>, University of Texas at Austin (Expected)</dd>
<dt>2012</dt>
<dd><strong>BS Biochemistry</strong>, <em>Magna Cum Laude</em>, University of Miami</dd>
</dl>

<h3>Current Affiliation</h3>
<dl class="cv">
<dt>2015 - present</dt>
<dd>Graduate Research Assistant, Wilke Lab, University of Texas at Austin</dd>
</dl>

<h3>Publications</h3>
<ul class="cv">
<li>
  <strong>B. R. Jack</strong>, D. R. Boutz, M. L. Paff, B. L. Smith, J. J. Bull, C. O. Wilke (2017). Reduced protein expression in a virus attenuated by codon deoptimization. G3: Genes, Genomes, Genetics, in press. 
  <br />DOI: <a href="http://dx.doi.org/10.1534/g3.117.041020">10.1534/g3.117.04102</a>&nbsp;&nbsp;
  <div data-badge-popover="right" data-badge-type="4" data-doi="10.1101/106799" data-hide-no-mentions="true" class="altmetric-embed"></div>
</li>
<li>
  <strong>B. R. Jack</strong>, A. G. Meyer, J. Echave, C. O. Wilke (2016). Functional sites induce long-range evolutionary constraints in enzymes. PLoS Biol 14(5): e1002452. 
  <br />DOI: <a href="http://dx.doi.org/10.1371/journal.pbio.1002452">10.1371/journal.pbio.1002452</a>&nbsp;&nbsp;
  <div data-badge-popover="right" data-badge-type="4" data-doi="10.1371/journal.pbio.1002452" data-hide-no-mentions="true" class="altmetric-embed"></div>
</li>
<li>
  E. L. Jackson, A. Shahmoradi, S. J. Spielman, <strong>B. R. Jack</strong>, and C. O. Wilke (2016). Intermediate divergence levels maximize the strength of structure–sequence correlations in enzymes and viral proteins. Protein Science, 25: 1341–1353.
  <br/>DOI: <a href="http://dx.doi.org/10.1002/pro.2920">10.1002/pro.2920</a>&nbsp;&nbsp;
  <div data-badge-popover="right" data-badge-type="4" data-doi="10.1002/pro.2920" data-hide-no-mentions="true" class="altmetric-embed"></div>
</li>
<li>
  <strong>B. R. Jack</strong>, S. P. Leonard, D. M. Mishler, B. A. Renda, D. Leon, G. A. Suárez, and J. E. Barrick (2015). Predicting the genetic stability of engineered DNA sequences with the EFM Calculator. ACS Synthetic Biology 4(8):939–943.
  <br/>DOI: <a href="https://dx.doi.org/10.1021/acssynbio.5b00068">10.1021/acssynbio.5b00068</a>&nbsp;&nbsp;
  <div data-badge-popover="right" data-badge-type="4" data-doi="10.1021/acssynbio.5b00068" data-hide-no-mentions="true" class="altmetric-embed"></div>
</li>
</ul>

<h3>Contributed Talks and Posters</h3>
<ul class="cv">
  <li>Disrupted proteome in a virus attenuated by codon deoptimization. SMBE 2017, Austin, TX.</li>
<li>Tuning translation to create evolutionarily stable viruses. BEACON Congress 2016, Michigan State University, East Lansing, MI.</li>
<li>Functional sites induce long-range evolutionary constraints in enzymes. SMBE 2016, Gold Coast, Australia. <strong>Walter M. Fitch Award finalist</strong>.</li>
<li>Evolutionary constraints in enzymes. Mechanisms of Protein Evolution III 2015: Origins, University of Colorado, Aurora, CO. <strong>Received Haldane Graduate Speaker Award</strong> for best talk given by a graduate student.</li>
<li>Active sites and enzyme evolution. BEACON Congress 2015, Michigan State University, East Lansing, MI.</li>
</ul>

<h3>Teaching</h3>
<dl class="cv">
  <dt>Fall 2017</dt>
  <dd>Graduate Teaching Assistant, Biostatistics, University of Texas at Austin</dd>
  <dt>Spring 2017</dt>
  <dd>Teaching Assistant, Introduction to RNA-sequencing, Big Data in Biology Summer School, University of Texas at Austin</dd>
  <dt>Fall 2016</dt>
  <dd>Teaching Assistant, Introductory and Intermediate Python Short Courses, University of Texas at Austin
<dt>Spring 2016</dt>
<dd>Graduate Teaching Assistant, Bioinformatics and Computational Biology, University of Texas at Austin</dd>
</dl>

<h3>Past Employment</h3>
<dl class="cv">
<dt>2012 - 14</dt><dd><strong>Sr Laboratory Technologist</strong>, HLA/Molecular Genetics, American Red Cross
<p>Matched potential donors and recipients for bone marrow transplants and platelet transfusions. This included collecting and processing data from the Luminex xMap platform to determine donor and recipient genotypes. I also analyzed genomic sequences from immune cells to monitor the ongoing success or failure of a bone marrow transplant.</p>
</dd>
</dl>

<h3>Computer Language Skills</h3>
<ul class="cv">
<li>Proficient in Python, R, C++</li>
<li>Working knowledge of HTML, CSS, Javascript</li>
</ul>
